cmd/manager/main.go

```diff
@@ -72,7 +62,7 @@ func main() {
 	// Create a new Cmd to provide shared dependencies and start components
 	mgr, err := manager.New(cfg, manager.Options{
 		Namespace:          namespace,
-		MetricsBindAddress: fmt.Sprintf("%s:%d", metricsHost, metricsPort),
+		MetricsBindAddress: monitoring.GetMetricsAddress(),
```

Comment:
We are not adding an argument to change metrics bind address from the container and default values https://github.com/kubernetes-sigs/controller-runtime/blob/b704f447ea7c8f7059c6665143a4aa1f6da28328/pkg/metrics/listener.go#L30 are good enough, let's just not set this so we use the default metrics bind address.

Reply:
not sure I got you. why do we not want to use a specific port, that we can also reference to on components.go?

Reply:
ok I change `monitoring.GetMetricsAddress()` to use `metrics.DefaultBindAddress` , is that alright?

Reply:
We don't need to pass anything and we don't need "GetMetricsAddress" since we are using controller-runtime's defaults.

Reply:
I can  remove it, but then how do you feel about the fact that we're already parsing the port on `GetMetricsPort` using  `DefaultBindAddress` ([here](https://github.com/kubevirt/cluster-network-addons-operator/pull/928/files#diff-abbf397a1a9176a1fd8d85bc268d3acf4d55fb789feabd1c2ae818395e26083eR69))?

My thinking was that if we're already using DefaultBindAddress exclicitly there then I should be consistent.. waiting to see what you say